### PR TITLE
Sprinkle in some const

### PR DESCRIPTION
**Pull request by tristan957**

### CODE REVIEW

**Comment by tristan957** [comment]
Depends on alen PR

**Review by alexttx — APPROVED** [review]

File: lib/cn/kvset.c [hunk 1]
```diff
@@ -1773,15 +1773,23 @@ kvset_get_nth_vblock_id(struct kvset *ks, uint32_t index)
...
+uint32_t
+kvset_get_nth_vblock_wlen(const struct kvset *ks, uint32_t index)
+{
+    const struct vblock_desc *vbd = lvx2vbd(ks, index);
+
+    return vbd ? vbd->vbd_wlen : 0;
+}
+
+const struct vblock_desc *
+kvset_get_nth_vblock_desc(const struct kvset *ks, uint32_t index)
```
[the remark this answers is not in the document]
Comment (tristan957) [reply lib/cn/kvset.c:1792]: I didn't change the base branch for whatever reason and I was too lazy to change it, but yeah I am not hitting merge until I rebase.

[the remark this answers is not in the document]
Comment (alexttx) [reply lib/cn/kvset.c:1792]: I've run into this before when I wanted to push up a series of dependent PRs.  You can create PRs based off other PR branches, so the diffs show up as you expect.  And you have to merge them in order of course.  But the workflow for the remaining PRs completely falls apart when you try to merge them.

[the remark this answers is not in the document]
Comment (tristan957) [reply lib/cn/kvset.c:1792]: Right. I will just ping for a re-review when I get the diff in order.

**Review by jjacob512 — APPROVED** [review]

**Pull request merged**